tests/manual/test-procedure.rst

```diff
@@ -298,6 +384,18 @@ Test Case: Zoom changes the Text Editor font size
 - Click the button with a magnifying lens and a ``-`` sign.
 - [ ] Confirm the font in the Text Editor has been decreased.
 
+Test Case: Connect over WebUSB and use REPL
+'''''''''''''''''''''''''''''''''''''''''''
+- [ ] Connect to micro:bit and confirm that menu now shows options to "Flash" and "Disconnect".
+- [ ] "Open Serial" and confirm you can enter the REPL by click or CTRL-C.
```

Comment:
Should there be a step before this one to flash the micro:bit?
In that case, the title would have to be renamed as well.